src/SprykerSdk/Zed/AppSdk/Business/Manifest/Builder/ManifestBuilder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Manifest\Builder;
+
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+
+class ManifestBuilder implements ManifestBuilderInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected $transferToManifestTypeMap = [
+        'int' => 'integer',
+    ];
+
+    /**
+     * @param \Generated\Shared\Transfer\ManifestRequestTransfer $manifestRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\ManifestResponseTransfer
+     */
+    public function addManifest(ManifestRequestTransfer $manifestRequestTransfer): ManifestResponseTransfer
+    {
+        $manifestResponseTransfer = new ManifestResponseTransfer();
+
+        $targetFilePath = $manifestRequestTransfer->getManifestPathOrFail();
+        $locale = $manifestRequestTransfer->getManifestOrFail()->getLocaleNameOrFail();
+
+        $targetFile = $targetFilePath . 'en_' . strtoupper($locale) . '.json';
+
+        $manifest = $this->addDefaults();
+
+        $manifest['name'] = $manifestRequestTransfer->getManifestOrFail()->getNameOrFail();
+        $manifest['provider'] = $manifestRequestTransfer->getManifestOrFail()->getNameOrFail();
+
+        $this->writeToFile($targetFile, $manifest);
+
+        return $manifestResponseTransfer;
+    }
+
+    /**
+     * @return array
```

Comment:
```suggestion
     * @return array<string, mixed>
```
Our tooling should show this error.

Reply:
I have added this.